commit 49ecf08114854d39ba3e4bdf038e26f1f498863b (refs/changes/67/3367/1)
Author: Mike Zeller <mike.zeller@joyent.com>
Date:   2018-02-14T15:50:33-08:00 (1 year, 8 months ago)
    
    TRITON-144 cloudapi is missing a "field" parameter in duplicate IP errors

diff --git a/lib/errors.js b/lib/errors.js
index 9a7c949..e74be82 100644
--- a/lib/errors.js
+++ b/lib/errors.js
@@ -438,6 +438,7 @@ function translateDuplicateIpErrors(error) {
              * have provisioned the zone.
              */
             return {
+                field: 'ip',
                 code: 'UsedBy',
                 message: 'IP in use'
             };
diff --git a/test/machines.test.js b/test/machines.test.js
index 848f19c..67c1380 100644
--- a/test/machines.test.js
+++ b/test/machines.test.js
@@ -1628,6 +1628,7 @@ test('Create Machine using network and in use IP', function (t) {
             code: 'InvalidParameters',
             message: 'Invalid parameters',
             errors: [ {
+                field: 'ip',
                 code: 'UsedBy',
                 message: 'IP in use'
             } ]
